RELEASE.md

```diff
@@ -0,0 +1,95 @@
+# 🦦 Dr. OtterAI Code Review, PhD - Release Notes
```

Comment:
Consider adding a brief introduction to the release notes to provide context for the reader. This can help set the stage for the features and improvements listed. 📖